src/Sulu/Bundle/MediaBundle/Resources/public/scss/modules/single-media-selection.scss

```diff
@@ -0,0 +1,153 @@
+.single-media-selection {
+    & {
+        height: 500px;
+
+        &, .slide, .overlay-footer {
+            width: 930px !important;
```

Comment:
Okay, this only affects the `.single-media-selection`, which is currently only used in the ckeditor plugin... We should maybe still try to get it running without these, only add these important styles in the ckeditor context. Can you maybe add a comment for the moment, and create an issue for now?